website/frontend/src/components/App/App.tsx

```diff
@@ -88,52 +101,52 @@ const AppComponent: FC = () => {
   };
 
   return (
-    <BrowserRouter>
-      <AlertProvider template={AlertTemplate} {...options}>
-        <div className="app">
-          {(isProgramUploading || isMessageSending) && (
-            <>
-              <div className="overlay" />
-              <LoadingPopup />
-            </>
+    <AlertProvider template={AlertTemplate} {...options}>
+      <div className="app">
+        {(isProgramUploading || isMessageSending) && (
+          <>
+            <div className="overlay" />
+            <LoadingPopup />
+          </>
+        )}
+        <Header />
+        <Main>
+          {isApiReady ? (
+            <Switch>
+              <Route exact path={[routes.main, routes.uploadedPrograms, routes.allPrograms, routes.messages]}>
+                <Programs />
+              </Route>
+              <Route exact path={routes.program}>
+                <Program />
+              </Route>
+              <Route exact path={routes.state}>
+                <State />
+              </Route>
+              <Route exact path={routes.editor}>
+                <EditorPage />
+              </Route>
+              <Route exact path={routes.notifications}>
+                <NotificationsPage />
+              </Route>
+              <Route exact path={[routes.privacyPolicy, routes.termsOfUse]}>
+                <Document />
+              </Route>
+            </Switch>
+          ) : (
+            <SimpleLoader />
           )}
-          <Header />
-          <Main>
-            {isApiReady ? (
-              <Switch>
-                <Route exact path={[routes.main, routes.uploadedPrograms, routes.allPrograms, routes.messages]}>
-                  <Programs />
-                </Route>
-                <Route exact path={routes.program}>
-                  <Program />
-                </Route>
-                <Route exact path={routes.state}>
-                  <State />
-                </Route>
-                <Route exact path={routes.editor}>
-                  <EditorPage />
-                </Route>
-                <Route exact path={routes.notifications}>
-                  <NotificationsPage />
-                </Route>
-                <Route exact path={[routes.privacyPolicy, routes.termsOfUse]}>
-                  <Document />
-                </Route>
-              </Switch>
-            ) : (
-              <SimpleLoader />
-            )}
-          </Main>
-          {isFooterHidden() || <Footer />}
-          <Alert />
-        </div>
-      </AlertProvider>
-    </BrowserRouter>
+        </Main>
+        {isFooterHidden() || <Footer />}
+        <Alert />
+      </div>
+    </AlertProvider>
   );
 };
 
 export const App = () => (
```

Comment:
Could you refactor this in the way one component per file?

Reply:
https://app.clickup.com/t/21zuycw